### PR TITLE
Add bit about going unversioned->versioned

### PR DESCRIPTION
**Pull request by Sushisource**
Wanted to add a bit about how you can go from unversioned to versioned queues/workers.

### CODE REVIEW

File: docs-src/concepts/what-is-worker-versioning.md [hunk 1]
```diff
@@ -229,3 +229,7 @@ You can also use this API `GetWorkerTaskReachability` directly from within langu
 
 Unversioned Workers refer to Workers that have not opted into the Worker Versioning feature in their configuration.
 They receive tasks only from Task Queues that do not have any version sets defined on them, or that have open workflows that began executing before versions were added to the queue.
+
+Migrating from an unversioned Task Queue (ie: A Task Queue only using unversioned workers) is as simple as adding a new
+default Build ID to that Task Queue, and then deploying Workers with that Build ID. Any open Workflows will continue to
+be processed by unversioned Workers, and any new Workflows will be processed by Workers with the new Build ID.
```
Comment (DJSanti) [inline docs-src/concepts/what-is-worker-versioning.md:235]:
```suggestion
To migrate from an unversioned Task Queue, add a new default Build ID to the Task Queue.
From there, deploy Workers with the same Build ID.
Open Workflows will continue to be processed by unversioned Workers; new Workflow Executions will be process by Workers with the new Build ID.
```

**Review by DJSanti — CHANGES_REQUESTED** [review]
Made some changes for easier flow.

**Comment by DJSanti** [comment]
@Sushisource Whenever you can, please make changes and update this branch.

File: docs-src/concepts/what-is-worker-versioning.md [hunk 2]
```diff
@@ -229,3 +229,7 @@ You can also use this API `GetWorkerTaskReachability` directly from within langu
 
 Unversioned Workers refer to Workers that have not opted into the Worker Versioning feature in their configuration.
 They receive tasks only from Task Queues that do not have any version sets defined on them, or that have open workflows that began executing before versions were added to the queue.
+
+To migrate from an unversioned Task Queue, add a new default Build ID to the Task Queue.
+From there, deploy Workers with the same Build ID.
+Open Workflows will continue to be processed by unversioned Workers; new Workflow Executions will be process by Workers with the new Build ID.
```
Comment (flossypurse) [inline docs-src/concepts/what-is-worker-versioning.md:235]:
```suggestion
Unversioned Workers will continue processing open Workflows, while Workers with the new Build ID will process new Workflow Executions.
```

**Review by flossypurse — APPROVED** [review]

**Review by rachfop — APPROVED** [review]

**Pull request merged**